.cascade-code/Chapman.edu/_cascade/formats/helpers.velocity

```diff
@@ -38,29 +37,19 @@ Master Version: Chapman.edu/_cascade/formats/helpers.velocity
     #set ($imageUrlInt =
     $elem.getChild('image').getChild('imageUrlInt').getChild('path').value )
     #set ($imageUrlExt =
-    $elem.getChild('image').getChild('imageUrlExt').value )
+    $elem.getChild('image').getChild('imageUrlExt').getChild('path').value )
     #set ($altText =
     $elem.getChild('image').getChild('altText').value)
     #set ($objectPositionX = $elem.getChild('image').getChild('objectPositionX').value.toLowerCase())
     #set ($objectPositionY = $elem.getChild('image').getChild('objectPositionY').value.toLowerCase())
-
-    #if ($imageType.contains("none"))
     
-    #elseif ($imageType == 'internal cascade image')
+    #if (!$imageType.contains("none"))
```

Comment:
@cbryant24 seems like it should be backwards compatible with existing widgets that use this helper, but have you tested? Thinking 3-col `contact-profile-widget`

Reply:
nevermind, seems like it should work